docs/user.md

```diff
@@ -651,7 +653,8 @@ source venv/bin/activate
 docker system prune -a
 make kind-up
 # Wait 10 minutes
-make test
+make kind-test
+racetrack plugin install https://github.com/TheRacetrack/plugin-python-job-type/releases/download/2.4.0/python3-job-type-2.4.0.zip http://localhost:7002
```

Comment:
Won't the python3 job type be needed for kind-test deployment tests? 

Another thing - how about adding the `racetrack plugin install ... ` to make kind-test (and perhaps other Makefile cmds) so that whenever developer runs it, he has a fully functional setup (like it was in my days ;) ). But not insisting, as it's just minor automation, and I'm not 100% sure if that makes sense with the flexible plugin architecture.

Reply:
The plugin is needed indeed. Good news, I've already added it to test_deploy.py so it's always being installed as a part of end-to-end test (either on docker-compose or kind). However the `racetrack plugin install` is rather needed before `racetrack deploy` command, I left it in order not to depend on `kind-test` result

Incidentally, I'm hoping to have fully functional setup easily too. I'm planning to improve that flow when doing https://github.com/TheRacetrack/racetrack/issues/37. Right now, it was more like a first step to make it working (and pass tests on CI). Stay tuned :)

Reply:
Ok, so `racetrack plugin install` is done also inside `kind-test`, but you don't know if that will always happen in the future, that's why you're adding it here just to be sure user will have it working. Fair enough.